gcp/src/gcpContext.test.ts

```diff
@@ -1,24 +1,25 @@
 import { GcpContext, GcpResponse } from ".";
 
 const runtimeArgs = [
-  {},
   {
-    eventId: "12344",
+    _readableState: {highWaterMark: "12344"},
```

Comment:
add some spaces between the { }'s make it { highWaterMark: "12344" },

Reply:
Done